File: ipification-sdk-android-java/app/src/main/java/com/ipification/demoapp/Constant.java
package com.ipification.demoapp;

import com.ipification.mobile.sdk.android.IPConfiguration;

public  class  Constant {
    private static final String HOST = IPConfiguration.getInstance().getHOST_NAME(); // stage or live


    public static String EXCHANGE_TOKEN_URL = HOST + "/auth/realms/ipification/protocol/openid-connect/token";
    public static String USER_INFO_URL = HOST + "/auth/realms/ipification/protocol/openid-connect/userinfo";

    public static String DEVICE_TOKEN_REGISTRATION_URL = "https://demo-api.ipification.com/register-device";

}

File: ipification-sdk-android-kotlin/app/src/main/java/com/ipification/demoapp/Constant.kt
package com.ipification.demoapp

class Constant {

    companion object Factory {
        // for demo only
        var EXCHANGE_TOKEN_URL = "https://stage.ipification.com/auth/realms/ipification/protocol/openid-connect/token"
        var USER_INFO_URL = "https://stage.ipification.com/auth/realms/ipification/protocol/openid-connect/userinfo"


        // for IM
        const val DEVICE_TOKEN_REGISTRATION_URL = "https://cases.ipification.com/merchant-service/register-device"
        const val AUTOMODE_SIGN_IN_URL = "https://cases.ipification.com/merchant-service/s2s/signin"


    }
}
